### DISCUSSION TITLE
Problem installing toffy_env and starting jupyter notebook

### DISCUSSION THREAD

**Discussion by spc27** [angelolab/toffy#423] [Q&A]
Hi everyone, 
I'm trying to install toffy on mac. When I want to set up the toffy_env I got this error and I cannot start the jupyter notebook. I'm quite new in using python so could someone please guide me on how to solve this?
<img width="1440" alt="Screenshot 2023-09-13 at 8 24 04" src="https://github.com/angelolab/toffy/assets/117159668/1e668984-576f-4a90-9f7e-f9207a67037d">

**Comment by srivarra** [comment]
@spc27 It looks like installing `jupyter_contrib_nbextensions` with `pip` instead of `conda` may fix this issue. I'll update the `environment.yml` file.

**Reply by spc27** [reply]
Thank you for reply @srivarra, but I'm still getting the same error
